.flake8

```diff
@@ -0,0 +1,10 @@
+[flake8]
+# Line break occurred before a binary operator (W503)
+# The recent research unearthed recommendations by Donald Knuth to break before
+# binary operators. flake8 will soon follow up the new recommendation. Until
+# the flake8 update, let's ignore the check.
+#
+# https://www.flake8rules.com/rules/W503.html
+extend-ignore = W503, E203
+
+max-line-length = 119
```

Comment:
What's the reason behind the max-line-length?

Reply:
> What's the reason behind the max-line-length?

The idea is inspired by [the coding style of `Django` project](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/):

```
We allow up to 119 characters as this is the width of GitHub code review; anything longer requires horizontal scrolling which makes review more difficult.
```


My own user experience of `max-line-length = 119` is also good with modern wide monitor screen. Maybe let me quote the above statement of Django in our commit message (I think I did but maybe it was dropped accidentally when rebasing my history tree).

Reply:
Please feel free to let me know which line wrap number you prefer. 72, 79 or 119 all works for me.

Reply:
Ah, OK. I tend to use 79, but I'm fine with 119.